templates/federated_reporting/import_file.sh

```diff
@@ -22,8 +21,7 @@ hostkey=$(basename "$file" | cut -d. -f1)
 
 mv "$file" "$file.importing"
 
-  # temp step to see the result of sed filter
-  "$CFE_FR_EXTRACTOR" $CFE_FR_EXTRACTOR_ARGS "$file.importing" | sed $CFE_FR_SED_ARGS $(printf ' -f %s' $sed_scripts) > $file.after-sed
+"$CFE_FR_EXTRACTOR" $CFE_FR_EXTRACTOR_ARGS "$file.importing" | sed $CFE_FR_SED_ARGS $(printf ' -f %s' $sed_scripts) > $file.after-sed
```

Comment:
I struggle with this a bit. I really want the .after-sed file which shows what we actually imported but if the files are large then this will eat up disk space. Some ideas for debugability? Maybe add a DEBUG flag that could be set in def.json in case there is trouble? As it is only the LATEST dump will be left for the .after-sed and .log files so it won't grow "forever" just like the .failed files are right now.

Reply:
CFEngine can cull old files

Reply:
I will rework this to put everything back "inline" and not generate this .after-sed file to disk. We can debug problems by looking at the .log files and using the .failed dump file along with sed filters to get back to what I had in .after-sed which is the actual SQL run during the import.

Reply:
fixed. tested in aws. Looks good to me.